docker-arweave-server

```diff
@@ -2,7 +2,7 @@
 
 set -o errexit
 
-mkdir -p ebin logs blocks wallets txs blocks/enc priv data/genesis_txs
+mkdir -p ebin logs priv
```

Comment:
Why is this needed? All three exists in git.

Reply:
The Dockerfile which was supposed to be used with this script has a dedicated stage for running the server where it just copies some of the build artifacts from the other stage, without copying the whole git folder. Creating `ebin` and `priv` (unlike `logs`) looks unnecessary anyway but I would leave these scripts as-is until we specifically dedicate some time to update or remove them.

Reply:
✅